newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -264,6 +266,7 @@ export const initialPreferences = {
     useUndefinedVariablesInAutocompletion: false,
     useGDJSDevelopmentWatcher: true,
     eventsSheetUseAssignmentOperators: false,
+    eventsSheetZoomLevel: 14,
```

Comment:
Open question: do you think we should name this `eventsSheetFontSize`?

Reply:
I wondered a bit. We could use it to scale icons' sizes as well (not implemented in this PR) so the naming seems ok with that in mind.

Reply:
Just saw that @Bouh add a comment for this purpose. I will add it then